draft-irtf-cfrg-fiat-shamir.md

```diff
@@ -71,18 +71,24 @@ Where:
 
 # The Codec interface
 
+A codec is a collection of:
+- functions that map prover messages into the hash function domain,
+- functions that map hash outputs into verifier messages (of the desired distribution).
```

Comment:
What's an example of a verifier message?

Reply:
yea, thanks -- @cjpatton this is standard nomenclature: a verifier message is a.. message from the verifier.

See: 
<img width="657" height="338" alt="Screenshot 2025-09-26 at 18 11 04" src="https://github.com/user-attachments/assets/c6d61f10-f0ac-46f5-9ddc-9f14fff0f68f" />
In [Justin's book](https://people.cs.georgetown.edu/jthaler/ProofsArgsAndZK.pdf).

See also:
<img width="857" height="229" alt="Screenshot 2025-09-26 at 18 12 58" src="https://github.com/user-attachments/assets/566dab91-6de8-4168-be7d-0e5bf04d5a96" />
In [Ale''s book](https://snargsbook.org/)